puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -80,51 +83,50 @@ def am_master(cursor: psycopg2.extensions.cursor) -> bool:
     return not cursor.fetchall()[0][0]
 
 
-pg_args = {}
+def get_config(
+    config_file: configparser.RawConfigParser,
+    section: str,
+    key: str,
+    default_value: str = "",
+) -> str:
+    if config_file.has_option(section, key):
+        return config_file.get(section, key)
+    return default_value
+
+
+pg_args = {
+    "connect_timeout": "600",
+    "user": "zulip",
+    "dbname": "zulip",
+}
 
-# Path to the root of the Zulip codebase in production
-sys.path.insert(0, "/home/zulip/deployments/current")
-# Path to the root of the Zulip codebase in development
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../..")))
+USING_PGROONGA = False
 try:
+    # Case (1); we insert the path to the development root.
+    sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../..")))
+    # Case (2) could take this branch if it set up the venv first, but
+    # there is no reason to unnecessarily bring in the Django
+    # dependency; it will fall into the path below due the venv not
+    # being configured.
```

Comment:
I think I'd prefer for case 2 to share the logic with the development environment, i.e. using Django settings if available.

Reply:
This bit was done by @alexmv so I'm not actually quite sure what to do for this bit.

Reply:
The problem is this leaves case (3) totally untested.  We don't have CI tests for splitting the DB onto a separate host, so it's quite likely that it will rot or break.  As it sits now, both clauses are tested by CI, by developers or by CI.

Django is also not a lightweight import.  Base memory usage without Django is 22M; it's 33M with it.  An additional 11M is not much in the grand scheme of things, but we're really not using it here.

Reply:
I've pushed this as a separate commit in the series which we can take or drop.